building_dialouge_webapp/templates/pages/consumption_result.html

```diff
@@ -1,7 +1,235 @@
-{% extends "base_footer_button.html" %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>consumption result</title>
+    <style>
+        .energy-bar {
+            display: flex;
+            width: 66%;
+            margin-left: auto;
+            margin-right: auto;
+            height: 50px;
+            border: 1px solid #ccc;
+            border-radius: 5px;
+            overflow: hidden;
+            font-family: Arial, sans-serif;
+        }
 
+        .energy-bar div {
+            flex: 1;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            color: white;
+            font-size: 16px;
+            font-weight: bold;
+        }
+
+        .section-a {
+            background-color: #00b300; /* Grün */
+        }
+
+        .section-b {
+            background-color: #99cc00; /* Hellgrün */
+        }
+
+        .section-c {
+            background-color: #ffcc00; /* Gelb */
+        }
+
+        .section-d {
+            background-color: #ff3300; /* Rot */
+        }
+
+        .container_house {
+            margin-left: auto;
+            margin-right: auto;
+            width: 66%;
+        }
+
+        .house{
+            position: relative;
+            left: {{house_position }};
```

Comment:
As CSS is out of hands in HTML, we have to set house position on SVG instead of CSS